Packs/RubrikPolaris/ReleaseNotes/1_0_2.md

```diff
@@ -0,0 +1,70 @@
+
+#### Integrations
+##### Rubrik Radar
+- Modified the graphql query during incident ingestion to filter on the 'lastUpdated' time for new incidents in accordance to backend functionality changes to Radar.  This also fixed a bug where new incidents were not being detected by XSOAR.
+- Added two new options in the Rubrik Polaris integration settings that allow the user to map the Radar Critical and Warning event severity levels to various XSOAR severity levels upon incident fetch.  The default severity mappings are currently:  Radar Critical -> XSOAR HIGH and Radar Warning -> XSOAR LOW.
+- Added the 'rubrik-cdm-cluster-connection-state' and 'rubrik-cdm-cluster-location' commands.  These commands have been added in as tasks to the Rubrik Post Intrusion Ransomware Investigation playbook.
+- Added the Rubrik CDM Cluster Location and Rubrik CDM Cluster Name custom fields.
+- Added functionality to pull in the Radar File Change attributes from Radar events during incident ingestion.
+- Various layout changes were made to the Rubrik Radar Incident Summary page.
+- Renamed the Rubrik Polaris CDM Cluster ID field to Rubrik CDM Cluster ID and modified the associated mapper.
+- Added the RubrikRadarFilesAdded, RubrikRadarFilesModified and RubrikRadarFilesDeleted scripts which pulls in the respective Radar File Change attributes.
+- Added the RubrikCDMClusterConnectionState script that pulls in the connection status for the CDM cluster.
+- Added the RubrikSonarTotalHits script to pull in the total number of Sonar Data Classification Hits.
+- Added the RubrikSonarOpenAccessFiles script to pull the open access permissions file count from Sonar.
+- Added the Rubrik Post Intrusion Ransomware Investigation playbook.  This has been made the default playbook for the Rubrik Radar incident type.
+- Added the Rubrik Polaris tab to the Rubrik Radar incident layout.  This new tab includes the Anomaly Detection Event Timeline and Data Classification sections.
+- Modified the Radar Mapper to now map the Hostnames field to the Radar Polaris "objectName" field of the Radar event.
+- Removed the Rubrik Analysis Progress, Rubrik Polaris Event Completed, and Rubrik Polaris Last Updated custom fields.
```

Comment:
These should be mentioned in their respective sections below. please remove from here.